docs/auth/README.md

```diff
@@ -0,0 +1,116 @@
+# Running Lighthouse on Authenticated Pages
+
+Standard usages of Lighthouse processes a page as a new user, with no previous session or storage data. This means that pages requiring authenticated access do not work without some additional setup. There are multiple ways to run Lighthouse on an authenticated page - this document focuses on the most flexible approach (Puppeteer), but mentions other approaches briefly at the end.
+
+If you just want to view the code, see `./example-lh-auth.js`.
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
+
+You can run this server locally if you like:
+
+```sh
+# be in root lighthouse directory
+yarn # install global project deps
+cd docs/auth
+yarn # install deps related to just this documentation
+yarn start # start the server on http://localhost:8000
+```
+
+## Puppeteer
+
+We will use Puppeteer to open a Chrome browser and setup our state by filling and submitting a login form. We'll designate a specific port for Chrome to open a debugging session, so that we can pass the same port to Lighthouse.
```

Comment:
I kind of want an outline of the whole process here that is worded more clearly of the whole process.

```suggestion
Puppeteer is a Browser automation tool that allows us to manipulate a browser session via code. We will use it to:
1. Create a new browser session.
2. Fill in a login form on a page.
3. Pass that authenticated session to Lighthouse.
```